dLars99/api-development-challenge
# Create route to read patients

# Description
Created two routes to read patients from the database: one to return the full list of patients, the other to return a single patient by Id.

Fixes #5 

## Type of Change
Enhancement

## Testing Steps
1. Run the app. Swagger should open with the new routes.
2. If you have not seeded your database, you will need to create some patients with the POST route
3. Use the Get route to retrieve the full list of patients
4. Use the Get(id) route to retrieve a single patient. An invalid result will return a Not Found status.
